src/substrait/textplan/tests/ParseResultMatchers.cpp

```diff
@@ -64,6 +65,44 @@ bool StringEqSquashingWhitespace(
   return atHave == have.end() && atExpected == expected.end();
 }
 
+std::vector<std::string> symbolNamesWithTypes(
+    const std::vector<std::shared_ptr<SymbolInfo>>& symbols,
+    const std::set<SymbolType>& types) {
+  std::vector<std::string> names;
+  for (const auto& symbol : symbols) {
+    if (types.find(symbol->type) != types.end()) {
+      names.push_back(symbol->name);
+    }
+  }
+  return names;
+}
+
+std::vector<std::string> OrderedSetDifference(
```

Comment:
```suggestion
std::vector<std::string> orderedSetDifference(
```

Or, if we are trying to be compatible with matchers in general, maybe change `symbolNamesWithTypes` to `SymbolNamesWithTypes`?

Reply:
Done.